Content.Server/Construction/ConstructionSystem.Initial.cs

```diff
@@ -97,61 +132,34 @@ private IEnumerable<EntityUid> EnumerateNearby(EntityUid user)
                 return null;
             }
 
-            var containers = new Dictionary<string, Container>();
```

Comment:
I'm _pretty_ sure this whole dynamic "containers" thing (and the "store:" attribute) doesn't need to exist anymore, now that we're validating everything first and then consuming all at once. But I could be wrong

Reply:
Gotcha, I didn't understand what the purpose actually was. The design doc is helpful, thanks for the link